src/app/page.tsx

```diff
@@ -13,6 +14,7 @@ export default function Home() {
       <SalonOwner />
       <NewOffer />
       <Services />
+      <Prices />
```

Comment:
The client unfortunately requested that the price list not be on the website. Remove the component import for now. We will keep your code as it may be useful in the future. Your work will not be in vain!